<launch>

  <arg name="subTopic" default="/image_raw"/>  
  <arg name="resize" default="true"/>
  <arg name="togray" default="true"/>
  <arg name="compress" default="false"/>
  <arg name="dstwidth" default="300"/>
  <arg name="dstHeight" default="150"/>

  <node pkg="cuda_imageprocessing" type="image_processing" name="image_processing" output='screen'/>
        <param name="subTopic"              type = "str"    value="$(arg subTopic)"/>
        <param name="resize"                type = "bool"   value="$(arg resize)"/>
        <param name="togray"                type = "bool"   value="$(arg togray)"/>
        <param name="compress"              type = "bool"   value="$(arg compress)"/>
        <param name="dstWidth"              type = "int"    value="$(arg dstwidth)"/>
        <param name="dstHeight"             type = "int"    value="$(arg dstHeight)"/>

</launch>
